packages/cli/tests/integration/git.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { tmpdir } from 'node:os';
+import { mkdir, mkdtemp, rm, writeFile } from 'node:fs/promises';
+import { dirname, join } from 'node:path';
```

Comment:
Pet peeve: unsorted imports (there's a vscode command for that!) ;)

Reply:
Can we configure ESLint to do enforce or fix this for us?